var/spack/repos/builtin/packages/py-csvkit/package.py

```diff
@@ -13,13 +13,17 @@ class PyCsvkit(PythonPackage):
     homepage = 'http://csvkit.rtfd.org/'
     url      = "https://pypi.io/packages/source/c/csvkit/csvkit-0.9.1.tar.gz"
 
+    version('1.0.4', sha256='1353a383531bee191820edfb88418c13dfe1cdfa9dd3dc46f431c05cd2a260a0')
     version('0.9.1', sha256='92f8b8647becb5cb1dccb3af92a13a4e85702d42ba465ce8447881fb38c9f93a')
 
```

Comment:
Missing deps:
```python
depends_on('py-argparse@1.2.1:', type=('build', 'run'), when='^python@:2.6')
depends_on('py-ordereddict@1.1:', type=('build', 'run'), when='^python@:2.6')
depends_on('py-simplejson@3.6.3:', type=('build', 'run'), when='^python@:2.6')
```

Reply:
I did not realize we were still supporting that. I will add those.

Reply:
These were added with latest commit.